.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Description
+Your description here.
+
+## Related Issues
+Closes #XXXX <- Issue number if existing
+
+## Checklist:
+<!-- Check these things before posting the pull request: -->
+- [ ]  Did you test the code?
+- [ ]  Was the Changelog Updated?
+- [ ]  Was the Documentation updated?
```

Comment:
Add `if needed`?

Or change to `Are all changes added to the Changelog?` and `Is the documentation up to date?` or similar.